api/routes/batch/resources.go

```diff
@@ -12,16 +12,17 @@ import (
 // Resources retrieves the resource ids of a batch
 func Resources(ctl monitor.BatchController) func(http.ResponseWriter, *http.Request) {
 	return func(w http.ResponseWriter, r *http.Request) {
+		var response Response
 		vars := mux.Vars(r)
-		batchID := vars["id"]
```

Comment:
I would have kept batchID and set response.Id juste before `response.Resources = resources`